### PR TITLE
feat: add filter-sidebar 

### LINKED ISSUE #5: added FilterSidebar
Here I added new component filter sidebar which containes various categories to sort the data.

### PR DESCRIPTION
**Pull request by Bhakti199**
Here I added filter sidebar to sort the data according the categories. #5 
Screenshot (1318)

### CODE REVIEW

File: src/Components/FilterSidebar/FilterSidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                <AiFillStar className="star-filter" />
+                to 4
+                <AiFillStar className="star-filter" />
+              </label>
+            </li>
+            <li className="flex-row sort-by-item">
+              <label>
+                <input type="radio" name="rating" />3{" "}
+                <AiFillStar className="star-filter" /> and below
+              </label>
```
Comment (mariyasada) [inline src/Components/FilterSidebar/FilterSidebar.jsx:98]: remove blank braces

**Review by mariyasada — APPROVED** [review]

**Review by Manju-sree — APPROVED** [review]
like the way you structured your code, using different folders and organizing it ever well and using Router is good to see. it's looks fine to merge.

File: src/Pages/ProductListingPage/ProductListingPage.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { FilterSidebar } from "../../Components/Index";
```
Comment (tushartiwari7) [inline src/Pages/ProductListingPage/ProductListingPage.jsx:2]: if you rename `Index` with `index` you don't even need to mention it  in the path.

File: src/App.js [hunk 3]
```diff
@@ -1,13 +1,17 @@
 import "./App.css";
 import { Navbar } from "./Components/Index";
 import { Home } from "./Pages/Home/Home";
+import { ProductListingPage } from "./Pages/ProductListingPage/ProductListingPage";
```
Comment (tushartiwari7) [inline src/App.js:4]: suggestion: make a index.js file /pages and export all Components from there.

**Review by tushartiwari7 — APPROVED** [review]
have a look at the suggestions given.

**Pull request merged**